sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubProperties.java

```diff
@@ -31,9 +31,10 @@
         final String[] partitionIds) {
         this.name = name;
         this.createdAt = createdAt;
+
         this.partitionIds = partitionIds != null
```

Comment:
partitionIds cannot be null and if it is, then should we bubble up an error?

Reply:
kk. will do